src/lib/Component/Dashboard/EzInfoTwigComponent.php

```diff
@@ -9,10 +9,10 @@
 namespace Ibexa\SystemInfo\Component\Dashboard;
 
 use Ibexa\Bundle\SystemInfo\SystemInfo\Value\IbexaSystemInfo;
-use Ibexa\Contracts\AdminUi\Component\Renderable;
+use Ibexa\Contracts\TwigComponents\ComponentInterface;
 use Twig\Environment;
 
-class EzInfoTwigComponent implements Renderable
+class EzInfoTwigComponent implements ComponentInterface
```

Comment:
We could think about finally deprecating that class name - or maybe we dont even have to care about it, at least for 5.0?
@adamwojs 
@ibexa/php-dev

Reply:
It's good idea. @ViniTou could you please open PR ?